content/news/010.open-tech-meetup-2.en.md

```diff
@@ -0,0 +1,19 @@
+---
+published: 2024-08-10
+description: Dresdnen's Network-Community comes together a second time.
+title: Second DD-IX Open Tech Meetup
+keywords: [ Event ]
+authors: [ Tassilo ]
+image: 010_open_tech_meetup_2.webp
+---
+
+The second Open Tech Meetup took place at TU Dresden at the beginning of August. The DD-IX team reported on current developments such as the PoP in City Center DC, new peers and our upcoming RIPE Labs article series.
+Of particular interest to network engineers reading this should be our announcement of the [DDNOG mailing list] (https://dd-ix.net/g/ddnog).
+After the half-hour presentations, the meetup was opened up for discussions on peering and network technologies.
+
+![Dinner](010_open_tech_meetup_dinner.webp)
+
+The discussions continued at Altes-Wettbüro where delicious food and wine were served to end the evening in a relaxed atmosphere.
```

Comment:
The discussion continued at Altes Wettbüro, where we enjoyed the beautiful weather and delicious food together.